test/accelerators/NNPA/backend/CMakeLists.txt

```diff
@@ -191,8 +191,8 @@ set(NNPA_TEST_LIST
     # ==OP== GlobalAveragePool
     # ==MIN== 1
     # ==LIM== - Input shape must be 4D tensor(NCHW).<br>- Dimensions in `H` and `W` must be static.
-    test_globalaveragepool_cpu,zdnn_meanreduce2d,NO_DYNAMIC_SHAPE_TEST
-    test_globalaveragepool_precomputed_cpu,zdnn_meanreduce2d,NO_DYNAMIC_SHAPE_TEST
+    test_globalaveragepool_cpu,zdnn_meanreduce2d,NO_DYNAMIC_SHAPE_TEST # Height and Width dimension must be static dimension.
+    test_globalaveragepool_precomputed_cpu,zdnn_meanreduce2d,NO_DYNAMIC_SHAPE_TEST # Height and Width dimension must be static dimension.
```

Comment:
Comments here look the same as `==LIM==`, are they necessary?

Reply:
Removed. Thanks.